app/src/main/java/fr/free/nrw/commons/MediaDataExtractor.java

```diff
@@ -61,8 +61,9 @@ public void fetch(String filename, LicenseList licenseList) throws IOException {
         }
 
         try{
-            Timber.d("Nominated for deletion: " + mediaWikiApi.pageExists("Commons:Deletion_requests/"+filename));
             deletionStatus = mediaWikiApi.pageExists("Commons:Deletion_requests/"+filename);
+            Timber.d("Nominated for deletion: " + deletionStatus);
+
         }
```

Comment:
@kamilya35231 I think only this line is enough to fix mentioned issue. Can you explain why you added other changes as well?

Reply:
I noticed that there were other redundancies in this file besides the one mentioned in the issue: - massive if-else statement which I simplified
- instead of using one parameter value directly, it was assigned to variable one and then used. I deleted intermediate variable

Thank you for the comment above, I am new to OSS and any feedback is very helpful. I'll add description next time :)

Reply:
It it always better to focus to solve related issue and create a separated PR for the other issues in the same file. The reason is we can easily find and revert related commits when something goes wrong. So can you please revert extra fixes, this pr should include only fix for the issue. Then we are done:)